docs/classes/Repository.md

```diff
@@ -182,7 +182,7 @@ found, does nothing.
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `id` | `string` | The ID of the [Entity](Entity.md) to set and expiration for. |
-| `ttlInSeconds` | `number` | THe time to live in seconds. |
+| `ttlInSeconds` | `number` | The time to live in seconds. |
```

Comment:
This is generated by TypeDoc from comments in the code. Fixing it here will not fix the problem. I have fixed this in another PR.